wrappers/python/examples/llama_training.py

```diff
@@ -103,114 +93,198 @@
+        hidden_size=model_torch.config.hidden_size,
+        hidden_size_tile=args.hidden_size_tile,
+        max_position_embeddings=model_torch.config.max_position_embeddings,
+        num_hidden_layers=model_torch.config.num_hidden_layers,
+        rms_norm_eps=model_torch.config.rms_norm_eps,
+        n_attention_head=model_torch.config.num_attention_heads,
+        num_key_value_heads=model_torch.config.num_key_value_heads,
+        intermediate_size=model_torch.config.intermediate_size,
+        intermediate_size_tile=args.intermediate_size_tile,
+        n_head_tile=args.n_head_tile,
+        dtype=args.dtype
+        )
+
+    print(llama_config_nntile)
+
+    single_batch_pos_ids = np.arange(args.seq_len).reshape(1, args.seq_len)
+    pos_ids = np.repeat(single_batch_pos_ids, args.minibatch_size, axis=0)
+
+    mask = np.array(np.triu(np.ones((args.seq_len, args.seq_len))),
+                        dtype=bool, order="F")
+    llama_nntile, next_tag = Llama_nntile.from_torch(model_torch,
+                                                    args.minibatch_size,
+                                                    args.minibatch_size_tile,
+                                                    args.seq_len,
+                                                    args.seq_len_tile,
+                                                    pos_ids,
+                                                    mask,
+                                                    llama_config_nntile,
+                                                    next_tag)
+    time1 = time.time() - time0
+    print("Converting PyTorch model to NNTile",
+          "requires {} seconds".format(time1))
+    del model_torch
+
+if args.nepochs > 0:
+    from pathlib import Path
+
+    import numpy as np
+    splitted_darasetfile = args.dataset_file.split("/")
+    if splitted_darasetfile[-1] == "train.bin":
+        train_data = np.memmap(Path(args.dataset_path) /
+                                args.dataset_file,
+                                dtype=np.uint16, mode='r')
+        train_tokens_raw = np.array(train_data, order='F', dtype=np.int64)
+        del train_data
+    else:
+        raise ValueError("Only train.bin file is accepted"
+                         "for training dataset!")
+
+    num_train_tokens = train_tokens_raw.shape[0]
+
+    num_train_seq = num_train_tokens // (args.seq_len + 1)
+    num_train_batches = num_train_seq // args.batch_size
+    num_train_tokens_truncated = num_train_batches * (args.batch_size
+            * (args.seq_len + 1))
+    train_tokens_trunc = np.array(
+        train_tokens_raw[:num_train_tokens_truncated],
+        order='F', dtype=np.int64)
+    train_tokens = train_tokens_trunc.reshape(num_train_batches,
+                                        num_minibatch,
+                                        args.minibatch_size,
+                                        args.seq_len + 1)
+
+if args.nepochs > 0:
```

Comment:
remove this if statement, as `nepochs` shall be checked if it is positive in the beginning of this script